packages/database/src/object-list.ts

```diff
@@ -10,33 +8,43 @@ async function getListsByCommunityId(communityId: string) {
   });
 }
 
-async function getCommunityListsWithObjects(communityId: string) {
+interface GetCommunityListsWithObjectsProps {
+  communityId: string;
+  limitObjects?: number;
+}
+
+async function getCommunityListsWithObjects({
```

Comment:
Sorry, I missed this one during a previous review: would it be possible to directly export the functions, without using an `export const objectList`? This way it is immediately clear which function can be used externally and which one cannot.

Reply:
I did this so I can import `objectList` and do, for example `objectList.getCommunityListsWithObjects()`. So, all the functions are grouped into one object. I thought grouping the individual functions would be nice if the database package grew.

Reply:
I understand. I find the names a bit hard to follow, though - e.g. `objectList.getCommunityListsWithObjects()` or `objectList.addObjectToList()` (lots of 'list' and 'object' :)).

If grouping of functions is important, would it be an idea to use a class instead? For example (pseudo code):

```
export class ObjectList {
  getByCommunityId(id, {withObjects: true}) // was: getListsByCommunityId and getCommunityListsWithObjects
  create() // was: createListForCommunity
  addObject() // was: addObjectToList
}
```